src/components/Battles.ts

```diff
@@ -326,6 +326,30 @@ export class Battles<TGameStartr extends FullScreenPokemon> extends Component<TG
     }
 
     /**
+     * Tests to see if all party Pokemon have fainted.
+     *
+     * @returns If a player's party is wiped or not.
```

Comment:
Elsewhere in the code base, the standard is to say `@returns Whether (thing).`, not `@returns If (thing) or not.`